mandatory/4-tax.js

```diff
@@ -17,7 +19,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  tax = calculateSalesTax(price);//calculating tax
+  //final_price = price + tax; //calculating final price
+ return `£${tax.toFixed(2)}`; 
+}
```

Comment:
That is right, you can easily format your value by using String interpolation, and you can use the function `toFixed()` to convert a number into specified number of digits.